src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  useMemo,
+  useState,
+  ChangeEvent,
+} from 'react';
+import {
+  useLocation,
+  useNavigate,
+  useSearchParams,
+} from 'react-router-dom';
+import { debounce } from 'lodash';
```

Comment:
you should add it to the dependency list in package.json
![image](https://user-images.githubusercontent.com/86923089/209831802-9a3dd1e3-a851-41a4-86eb-e03cb6c14050.png)